--- stack-v1.9.nix	2018-09-16 20:08:30.683765233 +0200
+++ stack.nix	2018-09-17 00:27:52.851814672 +0200
@@ -41164,7 +41164,7 @@
         ({ mkDerivation, aeson, attoparsec, base, bytestring, conduit
          , containers, directory, filepath, mtl, resourcet, scientific
          , stdenv, template-haskell, text, transformers
-         , unordered-containers, vector, yaml
+         , unordered-containers, vector, libyaml
          }:
          mkDerivation {
            pname = "yaml";
@@ -41178,13 +41178,13 @@
              filepath mtl resourcet scientific template-haskell text
              transformers unordered-containers vector
            ];
-           librarySystemDepends = [ yaml ];
+           librarySystemDepends = [ libyaml ];
            doHaddock = false;
            doCheck = false;
            homepage = "https://github.com/snoyberg/yaml#readme";
            description = "Support for parsing and rendering YAML documents";
            license = stdenv.lib.licenses.bsd3;
-         }) {inherit (pkgs) yaml;};
+         }) {inherit (pkgs) libyaml;};
       "yes-precure5-command" = callPackage
         ({ mkDerivation, base, Cabal, containers, mtl, parsec, random
          , stdenv
